[DNM] Cross-repo testing PR

MaxDesiatov opened the pull request:
<!-- What's in this pull request? -->
Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

<!--
If this pull request resolves any GitHub issues, link them.
For information about linking a pull request to an issue, see:
https://docs.github.com/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
-->
Resolves #NNNNN, fix apple/llvm-project#MMMMM.

<!--
Before merging this pull request, you must run the Swift continuous integration tests.
For information about triggering CI builds via @swift-ci, see:
https://github.com/apple/swift/blob/main/docs/ContinuousIntegration.md#swift-ci

Thank you for your contribution to Swift!
-->

MaxDesiatov commented:
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build linux toolchain

MaxDesiatov commented:
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain

MaxDesiatov commented:
@swift-ci Please test Amazon Linux 2 platform

MaxDesiatov commented:
@swift-ci Please test CentOS 7 platform

MaxDesiatov commented:
@swift-ci Please test Ubuntu 18.04 platform

MaxDesiatov commented:
https://github.com/apple/swift-tools-support-core/pull/415

@swift-ci test Apple Silicon

MaxDesiatov commented:
@swift-ci build toolchain

MaxDesiatov commented:
https://github.com/apple/swift-tools-support-core/pull/415

@swift-ci test Apple Silicon

MaxDesiatov commented:
@swift-ci build toolchain linux

MaxDesiatov commented:
@swift-ci build toolchain linux

MaxDesiatov commented:
@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov committed:
commit

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-libdispatch/pull/792
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-libdispatch/pull/792
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-libdispatch/pull/793
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-libdispatch/pull/793
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-libdispatch/pull/793
https://github.com/apple/swift-integration-tests/pull/78

@swift-ci build toolchain linux

ahoppen commented:
https://github.com/apple/swift-integration-tests/pull/129

@swift-ci Please Build Toolchain Amazon Linux 2

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/7722

@swift-ci build toolchain

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-foundation/pull/5052

@swift-ci test webassembly

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-foundation/pull/5052

@swift-ci test

MaxDesiatov commented:
https://github.com/apple/swift-corelibs-foundation/pull/5052

@swift-ci build toolchain

MaxDesiatov commented:
@swift-ci test webassembly

MaxDesiatov commented:
@swift-ci test webassembly

MaxDesiatov committed:
Log swiftpm bootstrap script output

MaxDesiatov commented:
@swift-ci smoke test linux

MaxDesiatov commented:
@swift-ci test linux

MaxDesiatov committed:
Flush build step log messages in `build_script_invocation.py`

This ensures that log messages are printed as soon as a corresponding build step starts.

MaxDesiatov commented:
@swift-ci test linux

MaxDesiatov commented:
@swift-ci smoke test

MaxDesiatov commented:
@swift-ci test linux

MaxDesiatov commented:
@swift-ci test webassembly

MaxDesiatov commented:
@swift-ci test webassembly

MaxDesiatov commented:
@swift-ci test webassembly

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/8558
@swift-ci build toolchain amazon linux 2

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/8570
@swift-ci test macos

MaxDesiatov commented:
https://github.com/swiftlang/llvm-project/pull/10970
@swift-ci build toolchain macos

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/8932

@swift-ci build toolchain amazon linux 2

bkhouri commented:
https://github.com/swiftlang/swift-package-manager/pull/8932
@swift-ci build toolchain amazon linux 2

bkhouri commented:
https://github.com/swiftlang/swift-package-manager/pull/8932
@swift-ci build toolchain amazon linux 2

bkhouri commented:
https://github.com/swiftlang/swift-package-manager/pull/8932
@swift-ci build toolchain amazon linux 2

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/8932
@swift-ci build toolchain amazon linux 2

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/9114
https://github.com/swiftlang/swift-integration-tests/pull/166
@swift-ci smoke test

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/166
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/166
@swift-ci smoke test linux

MaxDesiatov committed:
Merge branch 'main' into maxd/cross-repo-testing

MaxDesiatov commented:
https://github.com/swiftlang/swift-driver/pull/2027
https://github.com/swiftlang/swift-integration-tests/pull/167
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-driver/pull/2027
https://github.com/swiftlang/swift-integration-tests/pull/167
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-driver/pull/2027
https://github.com/swiftlang/swift-integration-tests/pull/167
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/167
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-sdk-generator/pull/237
@swift-ci build toolchain

MaxDesiatov commented:
@swift-ci smoke test

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/170
@swift-ci smoke test linux

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/171
@swift-ci smoke test

MaxDesiatov commented:
https://github.com/swiftlang/swift-integration-tests/pull/171
@swift-ci test windows

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/10002
@swift-ci smoke test

MaxDesiatov commented:
https://github.com/swiftlang/swift-package-manager/pull/10002
@swift-ci build toolchain

MaxDesiatov commented:
@swift-ci test macos

MaxDesiatov commented:
@swift-ci build toolchain
